src/main.rs

```diff
@@ -172,9 +93,7 @@ fn build(port: u16, outdir: &Path, sassbin: String) -> Result<()> {
 
         lazy_static!{
             static ref RE: regex::Regex = regex::Regex::new("([<>])").unwrap();
-        };
-
-        RE.replace_all(&path, "_").to_string()
+        }RE.replace_all(&path, "_").to_string()
```

Comment:
?? This formatting is weird

Reply:
don't ask me, blame jetbrains

Reply:
Then just don't autoformat, just remove this manually ฅ( ᵕ ω ᵕ )ฅ

```suggestion
        };
        
        RE.replace_all(&path, "_").to_string()
```